requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy>=1.22.0
-scipy~=1.10.1
-cirq~=1.1.0
+scipy>=1.5.0,<=1.10.0
```

Comment:
What is the reason for the upper bound `<=1.10.0` for `scipy`? And why is it smaller than `1.10.1` (current setting)?

Reply:
Good point. I think I was just testing stuff and used 1.10 for simplicity. I've bumped to the lastest working version for us in [`46b0237`](https://github.com/unitaryfund/mitiq/pull/1917/commits/46b02375640c4ed47b4a791525a49eed52c2532e).